cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/hvdc/Hvdc.java

```diff
@@ -0,0 +1,182 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.cgmes.conversion.elements.hvdc;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.powsybl.cgmes.conversion.elements.hvdc.IslandEndHvdc.HvdcEnd;
+import com.powsybl.cgmes.conversion.elements.hvdc.IslandEndHvdc.HvdcEndType;
+
+/**
+ *
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+class Hvdc {
+    List<HvdcEquipment> hvdcData;
+
+    Hvdc() {
+        this.hvdcData = new ArrayList<>();
+    }
+
+    void add(TPnodeEquipments tpNodeEquipments, IslandEndHvdc islandEndHvdc1, IslandEndHvdc islandEndHvdc2) {
+        islandEndHvdc1.hvdc.forEach(h -> add(tpNodeEquipments, h, islandEndHvdc2));
+    }
+
+    private void add(TPnodeEquipments tpNodeEquipments, HvdcEnd hvdc1, IslandEndHvdc islandEndHvdc2) {
+        HvdcEnd hvdc2 = islandEndHvdc2.selectSimetricHvdcEnd(hvdc1);
+        if (hvdc2 == null) {
+            return;
+        }
+        HvdcEndType type = hvdc1.computeType();
+        switch (type) {
```

Comment:
Add a `default` case throwing an exception

Reply:
It is an enum variable. It is not necessary.